mia/commands/definition.py

```diff
@@ -31,7 +31,15 @@
 import sys
 import zipfile
 import distutils.dir_util
-from urllib.request import urlretrieve
+
+try:
+    # For Python 3.
+    from urllib.request import urlretrieve
+except ImportError:
+    # For Python 2.
+    from urllib import urlretrieve
```

Comment:
Yeap, we can make the change sometime in the future, it would simplify our lives... and... I don't intend to write the code in order to add support for various M$ c**p OSes... :smiling_imp: